mongodb/map.jinja

```diff
@@ -1,25 +1,33 @@
-{% set mongodb = salt['grains.filter_by']({
-    'default': {
-      'key': '7F0CEB10',
-      'keyserver': 'keyserver.ubuntu.com',
-      'pkgs': [
-        'mongodb-org',
-      ],
-      'pip_pkgs': [
-        'pymongo',
-      ],
-      'cluster_key_file': '/etc/mongodb_key',
-      'bind_ip': '127.0.0.1',
-      'port': 27017,
-      'enable_journal': True,
-      'install_pkgrepo': true,
-      'dbpath': '/var/lib/mongodb',
-      'logpath': '/var/log/mongodb/mongodb.log',
+{% set version = salt.pillar.get('mongodb:version', '3.4') %}
+{% set vagrant_host = salt.pillar.get('mongodb:vagrant_host', '192.168.33.10') %}
+{% set mongodb = salt.grains.filter_by({
+  'default': {
+    'pkgs': [
+      'mongodb-org',
+    ],
+    'pip_pkgs': [
+      'pymongo',
+    ],
+    'cluster_key_file': '/etc/mongodb_key',
+    'bind_ip': '127.0.0.1',
+    'port': '27017',
+    'enable_journal': True,
+    'install_pkgrepo': true,
+    'dbpath': '/var/lib/mongodb',
+    'logpath': '/var/log/mongodb/mongodb.log',
+  },
+  'Debian': {
+    'key': '0C49F3730359A14518585931BC711F9BA15703C6',
+    'keyserver': 'keyserver.ubuntu.com',
```

Comment:
Yep Debian and Ubuntu have the same key:
https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/
https://docs.mongodb.com/manual/tutorial/install-mongodb-on-debian/